woocommerce/woocommerce-gutenberg-products-block
# Enable Stylelint for the entire project #5262

This PR enables `Stylelint` for the entire project (now it is enabled only for the `/assets` folder).


Fixes #5262

### Testing

### Manual Testing

How to test the changes in this Pull Request:

Check out this branch

1. Break a stylelint rule in a .scss file that it is outside the folder `/assets`
2. Run the command: `npm run stylelint`
3. Check that you get an error from `Stylelint`
